models/ai87net-effnetv2.py

```diff
@@ -0,0 +1,92 @@
+        # Series of MBConv blocks
+        self.mb_conv1 = ai8x.MBConvBlock(32, 16, 3, stride=1, padding=0, bias=bias, se_ratio=None, expand_ratio=1,
+                                            fused=True, **kwargs)
+        self.mb_conv2 = ai8x.MBConvBlock(16, 32, 3, stride=1, padding=0, bias=bias, se_ratio=None, expand_ratio=4,
+                                            fused=True, **kwargs)
+        self.mb_conv3 = ai8x.MBConvBlock(32, 32, 3, stride=1, padding=0, bias=bias, se_ratio=None, expand_ratio=4,
+                                            fused=True, **kwargs)
+        self.mb_conv4 = ai8x.MBConvBlock(32, 48, 3, stride=1, padding=0, bias=bias, se_ratio=None, expand_ratio=4,
+                                            fused=True,**kwargs)
+        self.mb_conv5 = ai8x.MBConvBlock(48, 48, 3, stride=1, padding=0, bias=bias, se_ratio=None, expand_ratio=4,
+                                            fused=True, **kwargs)
+        self.mb_conv6 = ai8x.MBConvBlock(48, 96, 3, stride=1, padding=0, bias=bias, se_ratio=None, expand_ratio=4,
+                                            fused=False, **kwargs)                                     
+        self.mb_conv7 = ai8x.MBConvBlock(96, 96, 3, stride=1, padding=0, bias=bias, se_ratio=None, expand_ratio=4,
+                                            fused=False, **kwargs) 
+        self.mb_conv8 = ai8x.MBConvBlock(96, 128, 3, stride=1, padding=0, bias=bias, se_ratio=None, expand_ratio=4,
+                                            fused=False, **kwargs) 
+        self.mb_conv9 = ai8x.MBConvBlock(128, 128, 3, stride=1, padding=0, bias=bias, se_ratio=None, expand_ratio=4,
+                                            fused=False, **kwargs)                                     
+        # Head Layer
+        self.conv_head = ai8x.FusedConv2dBNReLU(128, 1024, 1, stride=1, batchnorm='Affine', padding=0, bias=bias,
+                                            eps=1e-03, momentum=0.01, **kwargs)
+
+        # Final linear layer
+        self.avg_pooling = ai8x.AvgPool2d((16, 16))
+        #self.dropout = nn.Dropout(0.2)
+        self.fc = ai8x.Linear(1024, num_classes, bias=bias, wide=True, **kwargs)
+
+    def forward(self, x):  # pylint: disable=arguments-differ
+        """Forward prop"""
+        x = self.conv_stem(x)
+        x = self.mb_conv1(x)
+        x = self.mb_conv2(x)
+        x = self.mb_conv3(x)
+        x = self.mb_conv4(x)
+        x = self.mb_conv5(x)
+        x = self.mb_conv6(x)
+        x = self.mb_conv7(x)
+        x = self.mb_conv8(x)
+        x = self.mb_conv9(x)
+        x = self.conv_head(x)
+        x = self.avg_pooling(x)
+        #x = self.dropout(x)
+        x = x.view(x.size(0), -1)
+        x = self.fc(x)
+        return x
+
+def ai87effnetv2(pretrained=False, **kwargs):
+    """
+    Constructs EfficientNet v2 model.
+    """
+    assert not pretrained
+    return AI87EfficientNetV2(**kwargs)
+
+models = [
+    {
+        'name': 'ai87effnetv2',
```

Comment:
YAML in synthesis refers to arch as ai85effnetv2_n?

Reply:
I will fix it